sssd_test_framework/utils/sss_override.py

```diff
@@ -0,0 +1,420 @@
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.utils.fs import LinuxFileSystem
+
+__all__ = [
+    "SSSOverrideUtils",
+    "SSSOverrideUser",
+    "SSSOverrideGroup",
+]
+
+
+class SSSOverrideUtils(MultihostUtility[MultihostHost]):
+    """
+    Management of local override users and groups, using sss_override.
+    """
+
+    def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
+        """
+        :param host: Remote host instance.
+        :type host: MultihostHost
+        :param fs: Linux file system
+        :type fs: LinuxFileSystem
+        """
+        super().__init__(host)
+
+        self.cli: CLIBuilder = CLIBuilder(host.ssh)
+        self._users: list[str] = []
+        self._groups: list[str] = []
+        self.fs: LinuxFileSystem = fs
+
+    def teardown(self) -> None:
```

Comment:
This can be removed. Together with self._users and self._groups. It will be handled automatically by SSSDUtils by https://github.com/SSSD/sssd-test-framework/commit/b50970458306818169a15671edf0887382b7bf9e.